perllib/FixMyStreet/App/Controller/Admin/Users.pm

```diff
@@ -32,13 +32,11 @@ sub index :Path : Args(0) {
         my $user_rs = FixMyStreet::DB->resultset("User")->search({ id => \@uids });
         if ( $c->get_param('remove-staff') ) {
             foreach my $user ($user_rs->all) {
+                $user->remove_staff;
                 $user->update({
-                    from_body => undef,
                     email_verified => 0,
                     phone_verified => 0,
```

Comment:
So, for my education, why not move phone_verified, email_verified to remove_staff too?

Reply:
You can remove staff without unverifiying the user? that's what happens by the middle remove_staff call, when someone removes the body from a user account, but the account remains verified.